main/res/values/strings.xml

```diff
@@ -689,6 +689,8 @@
     <string name="init_trackautovisit">Visit TBs</string>
     <string name="init_summary_trackautovisit">Set trackables to \"Visited\" by default</string>
     <string name="init_sigautoinsert">Insert automatically</string>
+    <string name="init_offlinelogs_homescreen">Show offline logs</string>
+    <string name="init_summary_offlinelogs_homescreen">A counter like \"+1\" will be displayed on home screen if you have logs that are not yet submitted</string>
```

Comment:
Suggestion:
A offline log counter (e.g. \"+ 1\") will be shown next to your online find count on the home screen if you have logs, that are not submitted yet.